hoomd/md/pytest/test_meshbond.py

```diff
@@ -0,0 +1,246 @@
+# Copyright (c) 2009-2022 The Regents of the University of Michigan.
```

Comment:
I don't think they have to be pickable or loggable as the regular bond potentials tests aren't testing for it either.

Reply:
@SchoeniPhlippsn, we test both in `hoomd/md/pytest/test_potential.py` so that seems to more of an oversight than the expectation. Could you add them for these test, and I can make an issue for the bond tests.